wordpress-mobile/WordPress-iOS
# Notifications + Swift Mk 3: WPTableViewHandler

#### Details:

Reference #5613

#### To test:
1. Log into a Dotcom Account
2. Tap over the Notifications Tab
3. Verify that the Filters behave as expected
4. Verify that the Notification's Order doesn't differ from Develop's / Internal Beta build

Needs review: @aerych 
Eric, may i bother you with a WPTableViewHandler-Y Review?

Thanks in advance!
